sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,23 @@
+
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-table</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
```

Comment:
Take a look at the [pom file for azure-cosmos](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/pom.xml). I'm going to recommend that you copy lines 1-39, paste that over lines 1-14 in here, change the artifact and version to azure-cosmos-table version 1.0.0-beta.1 and that's a good start. You're also going to want to add this new library to the [eng/jacoco-test-coverage](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/jacoco-test-coverage/pom.xml) project and those lines you're pasting also [contain the properties to set the code coverage numbers](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/pom.xml#L37) to 0 or something low until you've got tests to generate coverage data.